src/graphics.rs

```diff
@@ -8,13 +8,17 @@ use crate::{
 use {
     anyhow::*,
     derivative::*,
+    image::Rgba,
+    image::RgbaImage,
     lyon::{
         math::*,
         tessellation::{self as t, FillOptions, StrokeOptions},
     },
     miniquad as mq,
     rlua::prelude::*,
     serde::{Deserialize, Serialize},
+    im::HashMap,
+    std::io::Error as IoError,
```

Comment:
as with above, `io::Error as IoError` should be included under the `std::{...}` tree as `std::{..., io::Error as IoError, ...}`. or, even just imported as `io` and then referred to with `io::Error` later. I avoid renaming types whenever possible